lib/ddtrace/buffer.rb

```diff
@@ -83,37 +83,37 @@ def measure_accept(trace)
       @buffer_accepted += 1
       @buffer_accepted_lengths += trace.length
     rescue StandardError => e
-      Datadog::Logger.log.debug("Failed to measure queue accept. Cause: #{e.message} Source: #{e.backtrace.first}")
+      Datadog.log.debug("Failed to measure queue accept. Cause: #{e.message} Source: #{e.backtrace.first}")
```

Comment:
I suppose it depends on whether you think of `log` as a noun or verb; the noun sense seems okay, verb maybe less so? `logger` is not ambiguous in the way, and is much more similar to what the class actually is.

One other thing to consider is line length: `log` is just a bit shorter. Personally, I don't really have a strong opinion, this can go either way and I'd be fine with it. @brettlangdon what do you think?

Reply:
I don't have a strong opinion here, I've used both, I think Python does:

```python
log = get_logger(__name__)
log.debug()
```

Reply:
Alright, I've adopted `logger`.